pkg/networkpolicy/controller.go

```diff
@@ -173,6 +176,24 @@ func newController(client clientset.Interface,
 
 	// reduce memory usage only care about Labels and Status
 	trim := func(obj interface{}) (interface{}, error) {
+		if po, ok := obj.(*v1.Pod); ok {
+			ips := make([]v1.PodIP, 0, len(po.Status.PodIPs))
+			for _, i := range po.Status.PodIPs {
+				ips = append(ips, v1.PodIP{IP: intern(i.IP)})
```

Comment:
Pod IPs don't get reused very quickly... This probably doesn't help _much_...
(Is there a performance tradeoff between "memory saved by interning" vs "extra allocations used in the process of interning"?)

Reply:
Yeah this is definitely likely to be dropped when I try to minimize the diff here. I did too much, mostly because I was misreading some pprofs before I added the explicitly `GC()` step. I doubt this adds any value.